google-cloud-compat-checker/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-compat-checker</artifactId>
+  <version>0.21.2-alpha-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>Google Cloud Java Compatibility Checker</name>
+  <url>https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-compat-checker</url>
+  <description>
+    Compatibility checker for Google Cloud Java.
+  </description>
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-pom</artifactId>
+    <version>0.21.2-alpha-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <site.installationModule>google-cloud-compat-checker</site.installationModule>
+    <artifact.version>${project.version}</artifact.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <version>2.0.3.Final</version>
+    </dependency>
+    <dependency>
+      <!-- It may look weird to have a compile dependency on a plugin,
+	   but we want to execute code from it at runtime, in order
+	   to not duplicate the logic ourselves -->
+      <groupId>kr.motd.maven</groupId>
```

Comment:
Sure, it's possible to get the information that way. *However*, the selection of netty-tcnative uses this logic. It seems both risky and wasteful to duplicate the logic, when all we want to do use get the exact result of the logic. Remember, we are trying to diagnose library loading errors here, so the more we diverge, the more likely we are to mismatch the actual library loading.